python-app/test_profile.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from flask_cors import CORS, cross_origin
+import pymongo
+from flask import request
+
+
+data = {}
+myclient = pymongo.MongoClient(
+    "mongodb+srv://admin:admin@cluster0-jacon.gcp.mongodb.net/test?retryWrites=true&w=majority")
+mydb = myclient["STUDYBUDDY"]
+mycollections = mydb["user_details"]
+
+data["name"] = "Test_Name"
+data["skills"] = "Test_Skills"
+data["courses"] = "Test_Courses"
+data["department"] = "Test_Department"
+
+
+def test_post_profile():
```

Comment:
use a test collection and don't test with the production collection.

Also, delete your collection after all the test cases have ran.

Reply:
It is a test collection. Please check

Reply:
delete this new collection after test run, otherwise  there'll always be an entry for name='Test_Name'